project/make.sh

```diff
@@ -58,6 +58,8 @@ DEFAULT_BUNDLES=(
 	cover
 	cross
 	tgz
+
+	fedora
```

Comment:
IMO, we should rename "ubuntu", since it's not accurate for what that file
is now, unless we fork it and make a "debian" too, but I'd much rather just
do all that as part of my larger ".deb generation" refactoring that's in
progress (because how we build ".deb" files right now is not ideal, let
alone sufficient for our needs anymore)